build.gradle

```diff
@@ -66,8 +66,8 @@ dependencies {
     implementation group: 'org.springframework' , name: 'spring-context', version: '5.3.7'
     implementation group: 'org.springframework' , name: 'spring-core', version: '5.3.7'
     implementation group: 'org.springframework.retry', name: 'spring-retry', version: '1.3.1'
-    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jdbc', version: '2.4.5'
-    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: '2.4.5'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jdbc', version: '2.6.6'
```

Comment:
For the other repos, it seems like we use a platform for the Spring dependencies. I wonder why they're explicit here. Is it safe to pull in TCL code referring to a different version of Spring than the service's?

Reply:
I think it's because we're only pulling in a few packages here since TCL is just a library (though it has a test app), rather than the full set to deploy a service.

I'm not actually sure how safe updating these packages out from under a service will be. My plan was to pull in the updated TCL version and upgrade Spring Boot to `2.6.6` at the same time for all the downstream services, so I didn't think about it very hard.